server/utils/datetimeUtils.ts

```diff
@@ -0,0 +1,50 @@
+const DATE_FORMAT_GB = new Intl.DateTimeFormat('en-GB', {
+  year: 'numeric',
+  month: '2-digit',
+  day: '2-digit',
+})
+
+class ParseResult {
```

Comment:
I think this would be better as a function.

```
const parseResult = (data?: string) => ({
  data,
  success: !!data,
})
```
and then the result type of the functions using it would be clearer and you wouldn't have to go to the class definition to figure out what it is and what it's doing.